hodor

```diff
@@ -155,15 +156,17 @@ class Hodor
         end
 
         # Initialize unison container
-        system("#{@docker_prefix} docker run #{share_volumes_opts.join(" ")} -p 45678:45678 -d --name fileshare gansbrest/fs-base")
+        system("#{@docker_prefix} docker run #{share_volumes_opts.join(" ")} -p 45678 -d --name #{@fileshare_name} gansbrest/fs-base")
       end
+      
+      fileshare_port = `#{@docker_prefix} docker port #{@fileshare_name} 45678 2>&1`.strip!.split(':')[1]
```

Comment:
seem like fileshare_port is duplicated in multiple places, is there a way to define it once?